routes/dao_coin_exchange_with_fees.go

```diff
@@ -1214,6 +1412,17 @@ func (fes *APIServer) HandleMarketOrder(
 		return nil, fmt.Errorf("HandleMarketOrder: Problem decoding public key %s: %v",
 			req.TransactorPublicKeyBase58Check, err)
 	}
+	// Create a transaction to whitelist the token in the user's wallet if it's not already,
+	// and if that's desired.
+	tokenWhitelistTxn, err := fes.MaybeCreateTokenWhitelistAssociation(
```

Comment:
can we wrap this in a `if !skipWhitelist`? I think we don't use this txn if skipWhitelist is true.